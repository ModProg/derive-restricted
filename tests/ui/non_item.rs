use derive_where::derive_where;

#[derive_where(Clone)]
const _: () = ();

fn main() {}
