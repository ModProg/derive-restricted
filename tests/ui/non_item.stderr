error: expected one of: `struct`, `enum`, `union`
 --> tests/ui/non_item.rs:4:1
  |
4 | const _: () = ();
  | ^^^^^
